server/src/test/java/org/opensearch/index/translog/transfer/TranslogTransferManagerTests.java

```diff
@@ -416,20 +421,27 @@ public void testDeleteStaleTranslogMetadata() {
         String tm2 = new TranslogTransferMetadata(1, 2, 1, 2).getFileName();
         String tm3 = new TranslogTransferMetadata(2, 3, 1, 2).getFileName();
         doAnswer(invocation -> {
-            ActionListener<List<BlobMetadata>> actionListener = invocation.getArgument(3);
+            ActionListener<List<BlobMetadata>> actionListener = invocation.getArgument(4);
             List<BlobMetadata> bmList = new LinkedList<>();
             bmList.add(new PlainBlobMetadata(tm1, 1));
             bmList.add(new PlainBlobMetadata(tm2, 1));
             bmList.add(new PlainBlobMetadata(tm3, 1));
             actionListener.onResponse(bmList);
             return null;
         }).when(transferService)
-            .listAllInSortedOrderAsync(eq(ThreadPool.Names.REMOTE_PURGE), any(BlobPath.class), anyInt(), any(ActionListener.class));
+            .listAllInSortedOrderAsync(
+                eq(ThreadPool.Names.REMOTE_PURGE),
+                any(BlobPath.class),
+                anyString(),
```

Comment:
nit: can we replace with eq(TranslogTransferMetadata.METADATA_PREFIX)

Reply:
ack , will address in a follow up PR .